templates/web/pages/user_form/user_form_page.tt.html

```diff
@@ -82,8 +82,8 @@
                     <label for="[% field.field %]">	[% field.label %]</label>
                     <input type="email" id="[% field.field %]" name="[% field.field %]" value = "[% field.value %]" autocomplete="email"/>
                 [% ELSIF field.type == 'checkbox' %]
-                    [% IF accepted_organization && section.id == "professional" %]
-                        <p>[% field.warning %]</p>
+                    [% IF accepted_organization && section.id == "professional" && user_org.defined %]
+                        <p>[% f_lang('this_is_a_pro_account_for_org', {'org' => user_org} ) %]</p>
```

Comment:
Hi @yuktea , have you checked if it works? (you can do that by creating an account with the "this is a professional account" checkbox, and then editing the account).
I think it won't work as is, because f_lang() expects a different format than sprintf  (%s vs something like <org>).
You can look at the strings in po/common/common.pot and po/common.en.po
The best solution is indeed to use f_lang(), but then you have to change the string. 

e.g. here is an example for another string:

# variable names between { } must not be translated
msgctxt "f_move_data_and_photos_to_main_language"
msgid "Move all data and selected photos in {language} to the main language of the product: {main_language}"
msgstr "Move all data and selected photos in {language} to the main language of the product: {main_language}"